app/assets/stylesheets/projects.scss

```diff
@@ -756,6 +762,45 @@ li.module-hover {
           }
         }
 
+        .actions {
+          align-items: center;
+          display: flex;
+          flex-basis: 1.5em;
+          height: 2em;
+          justify-content: center;
+
+          .project-actions-menu {
+            position: absolute;
+            right: .2em;
+            top: .2em;
+
+            a {
+              @include font-button;
+              padding: .5em 1em;
+
+              .fas {
+                margin-right: .4em;
+              }
+            }
+
+            &.new-comments::after {
+              @include red-dot;
+            }
+
+            &.new-comments {
+              .fa-comment {
+                position: relative;
+
+                &::after {
+                  @include red-dot;
+                  left: .9em;
+                  top: -.3em;
```

Comment:
Hm double include?

Reply:
One on card, one in dropdown list before comments.